nmostesting/suites/IS0502Test.py

```diff
@@ -593,3 +594,206 @@ def test_13(self, test):
+                                                             "for Sender {}".format(component["name"], resource["id"]))
+                                        sampling_error = False
+                                        if sampling == "4:4:4" or (sampling is not None and index == 0):
+                                            if component["width"] != flow["frame_width"] or \
+                                                    component["height"] != flow["frame_height"]:
+                                                sampling_error = True
+                                        elif sampling == "4:2:2":
+                                            if component["width"] != (flow["frame_width"] / 2) or \
+                                                    component["height"] != flow["frame_height"]:
+                                                sampling_error = True
+                                        elif sampling == "4:2:0":
+                                            if component["width"] != (flow["frame_width"] / 2) or \
+                                                    component["height"] != (flow["frame_height"] / 2):
+                                                sampling_error = True
+                                        if sampling_error:
+                                            return test.FAIL("Video Flow {} components do not match the expected "
+                                                             "dimensions for Sender sampling {}"
+                                                             .format(flow["id"], sampling))
+                            elif param_components[0] == "width":
+                                if flow["frame_width"] != int(param_components[1]):
+                                    return test.FAIL("Width for Sender {} does not match its Flow {}"
+                                                     .format(resource["id"], flow["id"]))
+                            elif param_components[0] == "height":
+                                if flow["frame_height"] != int(param_components[1]):
+                                    return test.FAIL("Height for Sender {} does not match its Flow {}"
+                                                     .format(resource["id"], flow["id"]))
+                            elif param_components[0] == "depth":
+                                for component in flow["components"]:
+                                    if component["bit_depth"] != int(param_components[1]):
+                                        return test.FAIL("Bit depth for Sender {} does not match its Flow {}"
+                                                         .format(resource["id"], flow["id"]))
+                            elif param_components[0] == "colorimetry":
+                                if flow["colorspace"] != param_components[1].split("-")[0].replace(".", ""):
```

Comment:
:thinking: :thought_balloon:

Reply:
Ha. I'll at the very least add a comment.